css/style.css

```diff
@@ -0,0 +1,80 @@
+*{
+    padding: 0;
+    margin: 0;
+    box-sizing: border-box;
+}
+main {
+    height: 100vh;
+    background-image: linear-gradient( 115deg, rgba(58, 58, 158, 0.8), rgba(136, 136, 206, 0.7) ),url(https://cdn.freecodecamp.org/testable-projects-fcc/images/survey-form-background.jpeg);
+    padding: 3%;
+    background-size:cover;
+    background-repeat: no-repeat;
+    display: flex;
+    justify-content: center;
+    flex-direction: column;
+    align-items: center;
+}
+
+.heading {
+    color: white;
+}
+
+#title {
+    font-family: Georgia, 'Times New Roman', Times, serif;
+}
+
+#description {
+    font-size: 20px;
+    font-family: cursive;
+}
+
+.form {
+    border-radius: 5px;
+    padding: 3%;
+    background-color: rgba(27, 27, 50, 0.8);
+}
+
+#name,#email, #number, #dropdown {
+    width: 100%;
+}
+
+label{
+    color: white;
+}
+
+#name-label {
+    color: white;
+    margin-bottom: 15px;
+    font-family: system-ui, -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
+}
+#email-label {
+    color: white;
+    margin-bottom: 15px;
+    font-family: system-ui, -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
+}
+#recommendation-section, #improve-section {
+    color: white;
+}
+
+#role-section {
+    color:white;
+}
+#number-label {
+    color: white;
+    margin-bottom: 15px;
+    font-family: system-ui, -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
+}
+
+#submit {
+    background-color: green;
```

Comment:
- [ ] Please make sure you're using the green color from your design template, it should be **#37af65** instead of #008000 (green)